microbenchmarks/src/main/java/io/camunda/microbenchmarks/msgpack/MsgpackBenchmark.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.microbenchmarks.msgpack;
+
+import io.camunda.zeebe.util.buffer.BufferUtil;
+import java.util.concurrent.TimeUnit;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
```

Comment:
🔧 I prefer seconds for readability, but I guess this depends on the test and how long it takes.

Reply:
See related https://github.com/camunda/camunda/issues/12241#issuecomment-1497232572

Reply:
Every test should have the approriate unif of measure, here it's usually less than a micro so..